mlbench_core/cli/cli.py

```diff
@@ -113,6 +114,13 @@ def cli(args=None):
 @click.option("--dashboard-url", "-u", default=None, type=str)
 def run(name, num_workers, gpu, light, dashboard_url):
     """Start a new run for a benchmark image"""
+    current_run_inputs = {}
+
+    try:
+        last_run_inputs = pickle.load(open("last_run_inputs.pkl", "rb"))
```

Comment:
Ideally, place it under `~/.local/share/mlbench`, using `os.path.join(os.environ['HOME'], ".local", "share", "mlbench")`